_posts/2021-04-29-WildFly2302-Released.adoc

```diff
@@ -0,0 +1,44 @@
+
+== WildFly Preview and Jakarta EE 9
+
+The link:https://docs.wildfly.org/23/WildFly_and_WildFly_Preview.html[WildFly Preview variant] of the server still passes the Jakarta EE 9 for the TCKs link:https://github.com/wildfly/certifications/blob/EE9/WildFly_23.0.2.Final/jakarta-full-platform.adoc[Full Platform] and the link:https://github.com/wildfly/certifications/blob/EE9/WildFly_23.0.2.Final/jakarta-web-profile.adoc[Web Profile].  We have previously sent a link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/326[compatibility certification request for the Full Platform] for WildFly Preview 23.0.1.Final and a link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/315[compatibility certification request for the Web Profile] for WildFly Preview 23.0.0.Final.
+
+Once released Jakarta EE 9.1 will be the first Jakarta EE release that allows a server to certify using both Java 8 and Java 11 for both the web profile and the full platform.
+
+For the various permutations we have published the following test results using the link:https://docs.wildfly.org/23/WildFly_and_WildFly_Preview.html[WildFly Preview variant] of the server with the candidate release of the Jakarta EE 9.1 TCKs and submitted corresponding certification requests to Eclipse.
+
+ * link:https://github.com/wildfly/certifications/blob/EE9.1/WildFly_23.0.2.Final/jakarta-web-profile-jdk8.adoc[Test Results - Web Profile - Java 8] / link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/349[Certification Request - Web Profile - Java 8]
+ * link:https://github.com/wildfly/certifications/blob/EE9.1/WildFly_23.0.2.Final/jakarta-web-profile.adoc[Test Results - Web Profile - Java 11] / link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/347[Certification Request - Web Profile - Java 11]
+ * link:https://github.com/wildfly/certifications/blob/EE9.1/WildFly_23.0.2.Final/jakarta-full-platform-jdk8.adoc[Test Results - Full Platform - Java 8] / link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/348[Certification Request - Full Platform - Java 8]
+ * link:https://github.com/wildfly/certifications/blob/EE9.1/WildFly_23.0.2.Final/jakarta-full-platform.adoc[Test Results - Full Platform - Java 11] / link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/346[Certification Request - Full Platform - Java 11]
+
+We are hopeful this will mean the link:https://docs.wildfly.org/23/WildFly_and_WildFly_Preview.html[WildFly Preview variant] of the server can be declared compatible with Jakarta EE 9.1 at the time it is released for the web profile and full platform against both Java 8 and Java 11.
```

Comment:
`web profile and full platform` should be `Web Profile and Full Platform`.

Reply:
Thanks, done